diff --git a/node_modules/react-native-autoheight-webview/autoHeightWebView/index.js b/node_modules/react-native-autoheight-webview/autoHeightWebView/index.js
index 3d0dc75..1cc31c2 100644
--- a/node_modules/react-native-autoheight-webview/autoHeightWebView/index.js
+++ b/node_modules/react-native-autoheight-webview/autoHeightWebView/index.js
@@ -1,11 +1,11 @@
-import React, {useState, useEffect, forwardRef} from 'react';
+import React, { useState, useEffect, forwardRef } from 'react';
 
-import {StyleSheet, Platform} from 'react-native';
+import { StyleSheet, Platform } from 'react-native';
 
-import {ViewPropTypes} from 'deprecated-react-native-prop-types';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 import PropTypes from 'prop-types';
 
-import {WebView} from 'react-native-webview';
+import { WebView } from 'react-native-webview';
 
 import {
   topic,
@@ -16,14 +16,19 @@ import {
 } from './utils';
 
 const AutoHeightWebView = React.memo(
-  forwardRef((props, ref) => {
-    const {
-      style,
-      onMessage,
-      onSizeUpdated,
-      scrollEnabledWithZoomedin,
-      scrollEnabled,
-    } = props;
+  forwardRef(({
+    style,
+    onMessage,
+    onSizeUpdated,
+    scrollEnabledWithZoomedin,
+    scrollEnabled,
+    showsVerticalScrollIndicator = false,
+    showsHorizontalScrollIndicator = false,
+    originWhitelist = ['*'],
+    viewportContent = Platform.OS === 'ios' ? 'width=device-width' : undefined,
+    scalesPageToFit = Platform.OS === 'android' ? false : undefined,
+    ...restProps
+  }, ref) => {
 
     const [size, setSize] = useState({
       height: style && style.height ? style.height : 0,
@@ -39,12 +44,12 @@ const AutoHeightWebView = React.memo(
             onMessage && onMessage(event);
             return;
           }
-          const {height, width, zoomedin} = data;
+          const { height, width, zoomedin } = data;
           !scrollEnabled &&
             scrollEnabledWithZoomedin &&
             setScrollable(!!zoomedin);
-          const {height: previousHeight, width: previousWidth} = size;
-          isSizeChanged({height, previousHeight, width, previousWidth}) &&
+          const { height: previousHeight, width: previousWidth } = size;
+          isSizeChanged({ height, previousHeight, width, previousWidth }) &&
             setSize({
               height,
               width,
@@ -62,9 +67,9 @@ const AutoHeightWebView = React.memo(
         ? scrollable
         : scrollEnabled;
 
-    const {currentSource, script} = reduceData(props);
+    const { currentSource, script } = reduceData({ ...restProps, style, viewportContent });
 
-    const {width, height} = size;
+    const { width, height } = size;
     useEffect(() => {
       onSizeUpdated &&
         onSizeUpdated({
@@ -74,7 +79,7 @@ const AutoHeightWebView = React.memo(
     }, [width, height, onSizeUpdated]);
 
     return React.createElement(WebView, {
-      ...props,
+      ...restProps,
       ref,
       onMessage: handleMessage,
       style: [
@@ -88,9 +93,14 @@ const AutoHeightWebView = React.memo(
       injectedJavaScript: script,
       source: currentSource,
       scrollEnabled: currentScrollEnabled,
+      showsHorizontalScrollIndicator,
+      showsVerticalScrollIndicator,
+      originWhitelist,
+      scalesPageToFit,
+      viewportContent,
     });
   }),
-  (prevProps, nextProps) => !shouldUpdate({prevProps, nextProps}),
+  (prevProps, nextProps) => !shouldUpdate({ prevProps, nextProps }),
 );
 
 AutoHeightWebView.propTypes = {
@@ -114,24 +124,6 @@ AutoHeightWebView.propTypes = {
   source: PropTypes.object,
 };
 
-let defaultProps = {
-  showsVerticalScrollIndicator: false,
-  showsHorizontalScrollIndicator: false,
-  originWhitelist: ['*'],
-};
-
-Platform.OS === 'android' &&
-  Object.assign(defaultProps, {
-    scalesPageToFit: false,
-  });
-
-Platform.OS === 'ios' &&
-  Object.assign(defaultProps, {
-    viewportContent: 'width=device-width',
-  });
-
-AutoHeightWebView.defaultProps = defaultProps;
-
 const styles = StyleSheet.create({
   webView: {
     backgroundColor: 'transparent',
